AssetRipper.Library/Exporters/Audio/AudioEffectsDefinitions.cs

```diff
@@ -0,0 +1,84 @@
+namespace AssetRipper.Library.Exporters.Audio
+{
+	public static class AudioEffectDefinitions
+	{
+		private enum UnityEffectType
+		{
+			Attenuation = -2,
+			Send = -3,
+			Receive = -4,
+			DuckVolume = -5
+		}
+
+		// <see href="https://documentation.help/FMOD-Ex/FMOD_DSP_TYPE.html" />
```

Comment:
Use xml documentation so I get proper intellisense.
```xml
/// <summary>
/// <see href="https://documentation.help/FMOD-Ex/FMOD_DSP_TYPE.html" />
/// </summary>
```
Otherwise, this looks good. Enum member names are most likely not covered by copyright.